src/main/kotlin/com/jetbrains/snakecharm/codeInsight/completion/SmkRulesAndCheckpointsCompletionContributor.kt

```diff
@@ -52,7 +53,18 @@ private class SmkRuleNameReferenceCompletionProvider : CompletionProvider<Comple
             context: ProcessingContext,
             result: CompletionResultSet
     ) {
-        val variants = collectVariantsForElement(parameters.position)
+        val variants = collectVariantsForElement(
```

Comment:
Class names for reference providers don't match with implementation, e.g.:
* `SmkRuleNameReferenceCompletionProvider` - adds both rule and checkpoint names into completion
* `SmkRulesAndCheckpointsObjectsCompletionProvider` - adds rule or checkpoint, but not both

Reply:
done